### DISCUSSION TITLE
[DISCORD] Channel clutter

### DISCUSSION THREAD

**Discussion by GalaxyGamingBoy** [simplified-coding/simplified-coding#124] [Ιδέες / Σχόλια]
There are a lot of channels in the discord currently, my proposed solution is to create one big channel that uses a **forum type** structure with labels for each lesson.

**Comment by GalaxyGamingBoy** [comment]
CC: @simplified-coding/simplified-coding-team

**Comment by odysseuskir** [comment]
There already is one. The individual channels are in place for people who wish to simply chat about a lesson.

**Reply by GalaxyGamingBoy** [reply]
-1, Too confusing for new people. Make it one channel bruh

**Reply by GalaxyGamingBoy** [reply]
With that logic there should be multiple channel lesson too

**Reply by GalaxyGamingBoy** [reply]
Just a big course-discussion

**Reply by GalaxyGamingBoy** [reply]
Forum is better for this

**Reply by odysseuskir** [reply]
The server uses an onboarding system that asks which courses the user is or wants to complete, so they are only in the channels they are interested in.

**Reply by odysseuskir** [reply]
Also, please post your messages in one reply, so as not to spam the inbox of the members in the discussion.

**Reply by GalaxyGamingBoy** [reply]
I beg to differ, it makes administrating stuff difficult and I really don't see why having dead channels would benefit. And also every discipline that will happen will probably be a hello or how are you (that are out of the channels topic) and questions.

**Comment by Nikos1508** [comment]
YES, that's what I was saying. one big channel

**Comment by GalaxyGamingBoy** [comment]
And also like @odysseuskir was saying, don't jump into replying (esp you ody since you did what I did before) This is for meeting purposes

**Comment by odysseuskir** [comment]
Since 2-1 on the team approve this, I am open to the change.
